oai_agents/common/arguments.py

```diff
@@ -83,6 +83,9 @@ def get_arguments(additional_args=[]):
     parser.add_argument("--num-dynamic-advs-per-heatmap", type=int, default=2)
     parser.add_argument("--custom-agent-ck-rate-generation", type=int)
 
+    parser.add_argument('--for-evaluation', type=bool, default=False, help="The trained agents are used for evaluating other agents. Please note that seeds and h_dim are different when agents are trained for evaluating others.)")
+    parser.add_argument("--num-of-training-variants", type=int, default=4)
```

Comment:
Same for here

Reply:
if num_of_training_variants is 4, we generate 4  agents. This is currently only for population of self-play agents since we have not implemented them for other methods.

Reply:
reminder to replace num_SPs_to_train with this var as well